doc/modules/preprocessing_targets.rst

```diff
@@ -7,6 +6,71 @@
 Transforming the prediction target (``y``)
 ==========================================
 
+Transforming target in regression
+---------------------------------
+
+:class:`TransformedTargetRegressor` transforms the target before fitting a
+regression model and inverting back the prediction to the original space. It
+takes a regressor and a transformer as argument::
```

Comment:
It takes as an argument the regressor that will be used for prediction, and the transformer that will be applied to the target variable.